### ISSUE TITLE
[A11Y][Bug] default aria-live messages not overridden by ariaLiveMessages

### ISSUE THREAD

**Issue by dtarjeft** [JedWatson/react-select#5949]
## Bug

The [custom-aria live example in the documentation](https://react-select.com/advanced), when read with a screen reader, reads the custom message, then "1 of 11" -- if options are customized to be hidden, this is confusing for users. `aria-onFocus` should be overridable, not just prepend-able.
